ui/oopsyraidsy/oopsy_summary_list.ts

```diff
@@ -277,7 +279,7 @@ export class OopsySummaryList implements MistakeObserver {
 
       const damageElem = document.createElement('div');
       damageElem.classList.add('death-row-amount');
-      if (event.amountClass)
+      if (event.amountClass !== undefined && event.amountClass.length > 0)
```

Comment:
```suggestion
      if (event.amountClass !== undefined)
```

`amountClass` here is all internal and so shouldn't ever be empty.